docs/documents/LLM-models.md

```diff
@@ -0,0 +1,225 @@
+
+- Pricing - region Paris
+
+On-Demand and Batch pricing for text models
+
+| Amazon Titan models       | Input (Per 1M tokens) | Output (Per 1M tokens) |
+| ------------------------- | --------------------- | ---------------------- |
+| Amazon Titan Text Lite    | $0.2                  | $0.25                  |
+| Amazon Titan Text Express | $0.25                 | $0.788                 |
+
+- Benchmarking
+
+There is limited available data about benchmarking Amazon Titan Text models. One of the very few available tests was conducted within the  [AWS COMMUNITY](https://community.aws/content/2b3FiirF9gbFM1Q7w1CUCDBhSwn/testing).
+
+The existence of websites that compare generative AI models doesn't provide much information about performance either. For example, this [Amazon Titan vs. ChatGPT vs. GPT-4 Comparison Chart](https://sourceforge.net/software/compare/Amazon-Titan-vs-ChatGPT-vs-GPT-4/) is not particularly informative because the software hasn't been reviewed yet. Additionally, there are unreliable comparisons between older versions of generative AIs and the then-unreleased Bedrock.
+
+The general available data about Amazon Titan Text models comes from the official Amazon site, which provides information regarding  [documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/titan-models.html), usage, pricing, [demo](https://aws.amazon.com/bedrock/titan/), [prompt engeneering](https://d2eo22ngex1n9g.cloudfront.net/Documentation/User+Guides/Titan/Amazon+Titan+Text+Prompt+Engineering+Guidelines.pdf), and more.
+
+### 2. Claude
+
+- Developer: Anthropic
+
+- Parameters: over 130 bilion
+
+- Access: API
+
+Claude boasts a wide context size, allowing it to consider a large amount of surrounding text when generating responses.
+Depending on the specific model variant, Claude’s context window ranges from 100,000 to 200,000 tokens. This makes it excellent at parsing and analyzing long documents and strings of text.
+
+_When interacting with Claude, it’s best to provide long documents or text before instructions or user input. The model pays extra attention to text near the bottom of the prompt._
+
+Capabilities:
+
+\- Claude generates non-toxic, safe, and helpful output by following ethical rules set by the developers.
+
+\- Users can input up to 100,000 tokens, allowing them to analyze hundreds of pages of documents with a single query.
+
+Claude 3 model family is the next generation of Anthropic's LLMs. This family includes three state-of-the-art models in ascending order of capability: Claude 3 Haiku, Claude 3 Sonnet, and Claude 3 Opus.
+
+- Pricing
+
+Claude 3/ Claude 2.1 context window - 200,000 token
+Claude 2.0 / Claude Instant context window - 100,000 token
+
+| Claude models   | Input (Per 1M tokens) | Output (Per 1M tokens) |
+| --------------- | --------------------- | ---------------------- |
+| Claude 3 Opius  | $15                   | $75                    |
```

Comment:
**suggestion:** correct this to `Claude 3 Opus`

Reply:
done